Marlin/src/pins/stm32f4/pins_FYSETC_S6.h

```diff
@@ -41,17 +41,17 @@
 // EEPROM Emulation
 //
 #if NO_EEPROM_SELECTED
-  #define FLASH_EEPROM_EMULATION
+  //#define FLASH_EEPROM_EMULATION
```

Comment:
I wonder how many people changing the default will inconvenience...

I have a board without an EEPROM, but it is damaged. If only the first batch is missing them, perhaps it is fine to change it.

Reply:
It may be necessary to allow to choose the way of storing the state ("eeprom") in the configuration.
Ok, I really don't know how many people have boards with an unsoldered chip, I thought it was not in the photo simply because they photographed an engineering sample. It is present on my board (it is in the fall of 2019).